bin/hdfcoinc/pycbc_page_injtable

```diff
@@ -45,34 +73,42 @@ else:
             newsnr = pycbc.events.ranking.newsnr(snr, chisq)
             
             found_names += ["%s SNR" % ifo, "CHISQ", "NewSNR"]
-            found_cols += [snr, chisq, newsnr]
+            found_cols += [numpy.round(snr,2), numpy.round(chisq,2), numpy.round(newsnr,2)]
```

Comment:
This list should also include the `if time < 0: output = 'n/a'` flag for snr, chisq and newsnr, as the table at the moment contains a lot of snr values where there wasn't a trigger which contributed to the found injection coincidence

Reply:
The travel time can be negative. Right? Or do you mean. I should set snr, chisq and newsnr to be 'N/A' if travel time is 'N/A'

Reply:
So this will be if the "H1/time" value is -1, not the travel time. This will be the same as checking if travel time is n/a

Reply:
This is now reported as 'nan' rather than 'n/a', and gives nan for snr/schisq/newsnr where appropriate